### DISCUSSION TITLE
Terraform provider

### DISCUSSION THREAD

**Discussion by lukipro** [ory/.github#66] [Ideas]
One missing part in the Ory ecosystem is an official Terraform provider. Are there any plans to kick off a project to develop it?

**Comment by vinckr** [comment]
Hello @lukipro 
Thanks for your interest! This is something Ory indeed has plans to implement in the future, but no concrete roadmap as of now. 
If you are interested in commercial support and have a budget to push the implementation forward, please [reach out to the team directly](https://www.ory.sh/contact/)!

PS: there is a community contributed provider for Ory Hydra: [svrakitin/terraform-provider-hydra](https://github.com/svrakitin/terraform-provider-hydra)
